tests/framework/web/AssetBundleTest.php

```diff
@@ -271,3 +287,20 @@ class TestAssetCircleB extends AssetBundle
         'yiiunit\\framework\\web\\TestAssetCircleA'
     ];
 }
+
+class TestAssetPerFileOptions extends AssetBundle
+{
+    public $basePath = '@testWebRoot';
+    public $baseUrl = '@testWeb';
+    public $css = [
+        'default_options.css',
+        ['tv.css', 'media' => 'tv'],
+        ['screen_and_print.css', 'media' => 'screen, print']
+    ];
+    public $js = [
+        'normal.js',
+        ['defered.js', 'defer' => 'defer'],
```

Comment:
Actually I was just asking for my own knowledge :)
I just made the change into the testcase and fixed my CHANGELOG entry.

On Thu, Nov 19, 2015 at 7:48 AM, Dmitry Naumenko <notifications@github.com>
wrote:

> In tests/framework/web/AssetBundleTest.php
> <https://github.com/yiisoft/yii2/pull/10166#discussion_r45307257>:
>
> > @@ -271,3 +287,20 @@ class TestAssetCircleB extends AssetBundle
> >          'yiiunit\\framework\\web\\TestAssetCircleA'
> >      ];
> >  }
> > +
> > +class TestAssetPerFileOptions extends AssetBundle
> > +{
> > +    public $basePath = '@testWebRoot';
> > +    public $baseUrl = '@testWeb';
> > +    public $css = [
> > +        'default_options.css',
> > +        ['tv.css', 'media' => 'tv'],
> > +        ['screen_and_print.css', 'media' => 'screen, print']
> > +    ];
> > +    public $js = [
> > +        'normal.js',
> > +        ['defered.js', 'defer' => 'defer'],
>
> Will you change it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yiisoft/yii2/pull/10166/files#r45307257>.
>



-- 
Mehdi Achour